tutorials/pyroot/pyroot002_pythonizationDecorator.py

```diff
@@ -0,0 +1,141 @@
+## \file
+## \ingroup tutorial_pyroot
+## \notebook -nodraw
+## This tutorial shows how to use the @pythonization decorator to add extra
+## behaviour to C++ user classes that are used from Python via PyROOT.
+##
+## \macro_code
+## \macro_output
+##
+## \date November 2021
+## \author Enric Tejedor
+
+import ROOT
+from ROOT import pythonization
+
+# Let's first define a new C++ class. In this tutorial, we will see how we can
+# "pythonize" this class, i.e. how we can add some extra behaviour to it to
+# make it more pythonic or easier to use from Python.
+#
+# Note: In this example, the class is defined dynamically for demonstration
+# purposes, but it could also be a C++ class defined in some library or header.
+# For more information about loading C++ user code to be used from Python with
+# PyROOT, please see:
+# https://root.cern.ch/manual/python/#loading-user-libraries-and-just-in-time-compilation-jitting
+ROOT.gInterpreter.Declare('''
+class MyClass {};
+''')
+
+# Next, we define a pythonizor function: the function that will be responsible
+# for injecting new behaviour in our C++ class `MyClass`.
+#
+# To convert a given Python function into a pythonizor, we need to decorate it
+# with the @pythonization decorator. Such decorator allows us to define which
+# is our target class, i.e. which class we want to pythonize.
+#
+# On the other hand, the decorated function - the pythonizor - must accept
```

Comment:
There was no "one the one hand" :) 

```suggestion
# The decorated function - the pythonizor - must accept
```

Reply:
There does not need to be, it can be implied :)

E.g. https://ell.stackexchange.com/questions/41619/can-on-the-other-hand-be-used-without-its-on-the-one-hand

Reply:
Sure but there is no evident contraposition here, try substituting "in constrast" in the place of "on the other hand" (same meaning but stronger).

I assume at some point the paragraphs read "The decorator takes these parameters. On the other hand, the decorated function takes these other parameters", but it's not the case anymore.

Reply:
Ok I'll reword.